packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md

```diff
@@ -0,0 +1,75 @@
+
+* **Update User Attributes** &mdash; Updates the profiles of users assigned to the integration and syncs those changes to your downstream application. Profile changes made in your application are overwritten with their respective Okta profile values.
+
+* **Deactivate Users** &mdash; Automatically deactivates user accounts when they are unassigned in Okta or their Okta accounts are deactivated.
+
+    Okta can also reactivate the application account if the integration is later reassigned to a user in Okta.
+
+* **Exclude Username Updates** &mdash; Prevents your downstream application profile from overwriting the Okta user profile when using the profile push feature.
+
+* **Sync Password** &mdash; Ensures a user's application password is always the same as their Okta passwords or, alternatively, allows Okta to generate a unique password for the user. See [Synchronize passwords](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-password-sync-main) .
+
+* **Profile Attribute Mappings** &mdash; Use this portion of the page to edit attributes and mappings in the Profile Editor. See  the [attributes and mappings](/docs/guides/build-provisioning-integration/attribute-mapping/) section or [Work with Okta user profiles and attributes](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Directory_Manage_Profile_Attributes) in the Okta product documentation.
+
+## To Okta
+
+This screen contains settings for all information that flows from your application upstream into Okta.
+
+Click **Edit** to make changes to the following sections.
+
+* **General** &mdash; Use this section to schedule imports and dictate a username format for imported users. You can also define a percentage of acceptable assignments before the [import safeguards](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-eu-import-safeguard) feature is automatically triggered.
+
+    If the Okta username is overridden due to mapping from a provisioning-enabled app, the custom mapping appears in this section.
+
+* **User Creation & Matching** &mdash; Matching rules are used when importing users from all applications and directories that allow importing. Establishing matching criteria allows you to specify how an imported user should be defined as a new user or mapped to an existing Okta user.
+
+  * **Imported user is an exact match to Okta user if** &mdash; The matching criteria that establishes whether an imported user exactly matches an existing Okta user. Choose any combination from the list of options to establish your criteria. For the new imported user to be considered an exact match, each option that you select must be true. Note that if you choose the third option, the first and second choices are disabled.
+
+  * **Allow partial matches** &mdash; Partial matching occurs when the first and last name of an imported user match those of an existing Okta user, but the username or email address do not.
```

Comment:
do not -> don't?

Reply:
changed to:
"... but not the username or email address."